safeweb/http.go

```diff
@@ -274,6 +283,13 @@ func (s *Server) serveBrowser(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Security-Policy", s.csp)
 	w.Header().Set("X-Content-Type-Options", "nosniff")
 	w.Header().Set("Referer-Policy", "same-origin")
+	if s.SecureContext {
+		if s.StrictTransportSecurityOptions != "" {
+			w.Header().Set("Strict-Transport-Security", s.StrictTransportSecurityOptions)
+		} else {
+			w.Header().Set("Strict-Transport-Security", defaultStrictTransportSecurityOptions)
+		}
```

Comment:
Optional alternative:
```suggestion
		w.Header().Set("Strict-Transport-Security", cmp.Or(s.StrictTransportSecurityOptions, defaultStrictTransportSecurityOptions))
```